File: 프로그래머스/1/12977. 소수 만들기/소수 만들기.cs
using System;

class Solution
{
    public int solution(int[] nums)
    {
        int answer = 0;
        
        
        for(int i=0; i<nums.Length-2; i++)
        {
            for(int j=i+1; j<nums.Length-1; j++)
            {
                for(int k=j+1; k<nums.Length; k++)
                {
                   
                    int n = nums[i] + nums[j] + nums[k];
                   
                    int count=0;
                    for(int t=2; t<=n; t++)
                    {
                        if(n%t==0) count++;
                    }
                    
                    if(count==1) answer++;
                }
            }
        }
        return answer;
    }
}

File: 프로그래머스/1/12977. 소수 만들기/소수 만들기.cpp
#include <vector>
#include <iostream>
using namespace std;

int solution(vector<int> nums) {
    
    int answer = 0;
    for(int i = 0; i < nums.size()-2; i++)
    {
        for(int j = i+1; j < nums.size()-1; j++)
        {
            for(int k = j+1; k < nums.size();k++ )
            {
             int sum = nums[i] + nums[j] + nums[k];
             for(int l = 2; l < sum; l++)
             {
                 if(sum%l ==0)
                 {
                     break;
                 }
                 if(l == sum -1 ){
                     answer++;
                 }
             }
                
            }
        }
        
    }

    
    return answer;
}